pkg/handler/handler.go

```diff
@@ -70,18 +70,35 @@ func WebhookHandler(w http.ResponseWriter, r *http.Request, reconciler gitopscon
 				Items:    make([]gitopsv1alpha1.GitOpsConfig, len(list.Items)),
 			}
 
+			// The targetList.Items is initialized with an empty GitOpsConfig at index 0
+			// We are dropping this item to ensure we do not try to start a reconsiliation job for an "empty" GitOpsConfig resource
+			targetList.Items = targetList.Items[1:]
```

Comment:
is there a way to do this on line 70 instead add a new line of code?

Reply:
@seanmalloy good call, I updated the code to initialize the `targetList.Items` to an empty slice.